packages/open-next/src/plugins/resolve.ts

```diff
@@ -44,7 +46,11 @@ export function openNextResolvePlugin({
     name: "opennext-resolve",
     setup(build) {
       logger.debug(`OpenNext Resolve plugin for ${fnName}`);
-      build.onLoad({ filter: /core\/resolve.js/g }, async (args) => {
+      const filter = new RegExp(
+        escapeStringRegexp(path.join("core", "resolve.js")),
```

Comment:
If the issue is just the `/`, can't we just do a simple regex like that instead
```ts
/core(\/|\\)resolve.js/g
```

Reply:
I have removed the need for an extra dependency in favor of including a `\\` in the regex. You are right, it might not be worth adding a new project dependency just for this.